client/my-sites/domains/domain-management/settings/cards/name-servers-card.tsx

```diff
@@ -228,7 +255,7 @@ const NameServersCard = ( {
 					onCancel={ handleCancel }
 					onReset={ handleReset }
 					onSubmit={ handleSubmit }
-					submitDisabled={ isLoading() }
+					submitDisabled={ isLoading() || !! hasWpcomNameservers() }
```

Comment:
Is the double `!!` intended?

Reply:
Good catch, it was necessary for a previous version of that check I used, but now it's guaranteed to be a boolean so it's not needed